packages/daimo-api/src/landline/landlineClogMatcher.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  LandlineTransfer,
+  landlineTransferToOffchainTransfer,
+  landlineTransferToTransferClog,
+  TransferClog,
+  TransferSwapClog,
+} from "@daimo/common";
+import { DaimoChain } from "@daimo/contract";
+import { Hex } from "viem";
+
+export function addLandlineTransfers(
```

Comment:
this function does something quite fancy--i'd add a sentence or two comment re: how it combines landlineTransfers onto existing transfer clogs vs creating new ones for pending deposits